src/TRoom.cpp

```diff
@@ -1160,21 +1156,12 @@ void TRoom::auditExits(const QHash<int, int> roomRemapping)
             if (roomRemapping.contains(exitRoomId)) {
                 QString auditKey = QStringLiteral("audit.remapped_special_exit.%1").arg(exitName);
                 userData.insert(auditKey, QString::number(exitRoomId));
-                if (mudlet::self()->showMapAuditErrors()) {
-                    QString infoMsg = tr("[ INFO ]  - In room with id: %1 correcting special exit \"%2\" that\n"
-                                         "was to room with an exit to invalid room: %3 to now go\n"
-                                         "to: %4.")
-                                              .arg(id)
-                                              .arg(exitName)
-                                              .arg(exitRoomId)
-                                              .arg(roomRemapping.value(exitRoomId));
-                    mpRoomDB->mpMap->postMessage(infoMsg);
-                }
-                mpRoomDB->mpMap->appendRoomErrorMsg(id,
-                                                    tr(R"([ INFO ]  - Room needed correcting of special exit "%1" that was to room with an exit to invalid room: %2 to now go to: %3.)")
-                                                            .arg(exitName)
-                                                            .arg(exitRoomId)
-                                                            .arg(roomRemapping.value(exitRoomId)));
+                QString auditMessage = tr("[ INFO ]  - Room with ID %1 needed correcting of special exit \"%2\". "
+                                          "It was to room with an exit to invalid room with ID %3 to now go to: %4.") 
+                                          // This text looks bogus. Please clarify!
```

Comment:
@SlySven

Reply:
The exit was to a room id which is not valid but was identifiable - this is most likely for a negative or zero room id (remember that the `TRoom` class has, I can't recall if that has changed, used `-1` as meaning "no exit in this direction" - so anything else could identify a valid room Id even if we do not currently support negative room Ids) - it is getting converted to a valid available positive room id...

Reply:
It says: "Room has special exit %2 - it was to room with an exit to invalid room id" ?!